types/k6/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/k6",
-    "version": "0.49.9999",
+    "version": "0.50.9999",
```

Comment:
Can you please revert this particular change as this isn't necessary. And should be done when we *do* release v0.50.0

Reply:
Thank you for your comment.
I've fixed the above point.

Please confirm again.